gapic-common/lib/gapic/call_options/error_codes.rb

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Gapic
+  class CallOptions
+    ##
+    # @private
+    # The gRPC error codes and their HTTP mapping
+    #
+    class ErrorCodes
+      # @private
+      # See https://grpc.github.io/grpc/core/md_doc_statuscodes.html for a
+      # list of error codes.
+      ERROR_CODE_MAPPING = [
```

Comment:
Since this isn't actually used as a constant anywhere other than in this module, consider just creating a local variable for now. (i.e. Set `error_code_mapping` instead of `ERROR_CODE_MAPPING` and reference it as such below when building `ERROR_STRING_MAPPING`.)

Reply:
:+1: